README.md

```diff
@@ -1,83 +1,82 @@
-# Sections Utility Tool
+# Canvas Course Manager
 
-## Build Directions
- 
-1. sectionsTool$ <code>mvn clean install</code>
-2. Copy to tomcat/webapp
-3. Add the property file on linux box <code>sectionsToolPropsSecure.properties</code>,   
-then in JAVA_OPTS add the  
-<code>-DsectionsToolPropsPathSecure=file:/file-path/sectionsToolPropsSecure.properties</code><br/>
-<code>-DsectionsToolFriendPropsPathSecure=file:/file-path/sectionsToolFriend.properties</code><br/>
-<code>-DsectionsToolFriendPropsPathSecure=file:/file-path/sectionsToolFriendSecure.properties</code><br/>
+## Project Description 
+  
```

Comment:
The description is a bit off. The SC UI will allow drag and drop, but the LTI will function via clicks (because of a11y).

All else is great.